### PR TITLE
[Bug 1248594] Use MLS for GeoIP detection

### PR DESCRIPTION
**Pull request by brittanystoroz**
Switching over from geo.mozilla.org to MLS for geoIP detection as outlined in [bug 1248594](https://bugzilla.mozilla.org/show_bug.cgi?id=1248594)

### CODE REVIEW

File: kitsune/sumo/templates/base.html [hunk 1]
```diff
@@ -237,10 +237,6 @@
 {% endif %}
 {# End Optimizely #}
 
-{% if include_geoip %}
```
Comment (mythmon) [inline kitsune/sumo/templates/base.html]: I think `include_geoip` is now unused, and can be removed. It is defined [here](https://github.com/mozilla/kitsune/blob/master/kitsune/sumo/context_processors.py#L23-L29).

**Comment by mythmon** [comment]
This is so much nicer. I'm glad they did the new API as JSON instead of JS.

**Comment by mythmon** [comment]
Can we add a comment [like Snippets did](https://github.com/mozilla/snippets-service/pull/143/files#diff-3adb3669370af06c37adc916ba7b7c3cR348) asking people not to directly use our key?

**Comment by brittanystoroz** [comment]
@mythmon this has been updated with the changes you requested & travis is happy :)

**Comment by safwanrahman** [comment]
@brittanystoroz remove the past commits that are made by others.

**Comment by brittanystoroz** [comment]
whoa, must have messed up my rebase. thanks for noticing @safwanrahman

**Comment by mythmon** [comment]
Wooh, thanks!

**Pull request merged**